### PR TITLE
Fix the provisioning which was failing if the venv directory did not exist

### PR DESCRIPTION
**Pull request by b4hand**
Also, a few miscellaneous clean-ups to the packaging and the Vagrant image.

### CODE REVIEW

File: scripts/vagrant/provision.sh [hunk 1]
```diff
@@ -32,12 +32,17 @@ pushd /vagrant
 
     # Install a virtualenv
     pip install virtualenv
-    if [ -d venv ]; then
+    if [ ! -d venv ]; then
```
Comment (sistawendy) [inline scripts/vagrant/provision.sh]: Clearly I didn't understand how `virtualenv` works.

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,9 @@
 *.pyc
+*.so
+.coverage
+.vagrant
+MANIFEST
```
Comment (sistawendy) [inline .gitignore]: What's this?

Comment (b4hand) [inline .gitignore]: MANIFEST is a generated file that is part of the egg that is uploaded to PyPi.

They are not supposed to be checked in:

http://stackoverflow.com/q/29836718/

File: scripts/vagrant/provision.sh (hunk 1 above)
Comment (b4hand) [inline scripts/vagrant/provision.sh]: `git blame` says the typo has been there since the file was added.

**Pull request merged**